doc/source/whatsnew/v1.0.0.rst

```diff
@@ -1133,6 +1133,7 @@ Reshaping
 - Bug in :func:`merge_asof` merging on a tz-aware ``left_index`` and ``right_on`` a tz-aware column (:issue:`29864`)
 - Improved error message and docstring in :func:`cut` and :func:`qcut` when `labels=True` (:issue:`13318`)
 - Bug in missing `fill_na` parameter to :meth:`DataFrame.unstack` with list of levels (:issue:`30740`)
+- Bug in :func:`unstack` can take tuple names in MultiIndexed data (:issue:`19966`)
```

Comment:
<code>:func:\`unstack\`</code> doesn't exist, so instead should probably be "<code>:meth:\`DataFrame.unstack\`</code> and <code>:meth:\`Series.unstack\`</code>"

Reply:
thanks! changed!